framework/src/wifi_manager/wifi_manager.c

```diff
@@ -1372,8 +1381,8 @@ wifi_manager_result_e wifi_manager_get_info(wifi_manager_info_s *info)
 	LOCK_WIFIMGR;
 	uint8_t *ip = (uint8_t *)&g_manager_info.ip4_address;
 	snprintf(info->ip4_address, 18, "%d.%d.%d.%d", ip[0], ip[1], ip[2], ip[3]);
-	memcpy(info->ssid, g_manager_info.ssid, 32);
-	memcpy(info->mac_address, g_manager_info.mac_address, 6);
+	memcpy(info->ssid, g_manager_info.ssid, WIFIMGR_SSID_LEN + 1);
```

Comment:
"WIFIMGR_SSID_LEN + 1" or "WIFIMGR_MACADDR_LEN"

Reply:
Here, SSID (info->ssid) should be the length of WIFIMGR_SSID_LEN + 1.
And as below, MAC addr (info->mac_address) should be the length of WIFIMGR_MACADDR_LEN.